SanitizeProfileEmail/main.ts

```diff
@@ -0,0 +1,27 @@
+import * as H from "@pagopa/handler-kit";
+import { azureFunction } from "@pagopa/handler-kit-azure-func";
+
+import {
+  PROFILE_COLLECTION_NAME,
+  ProfileModel
+} from "@pagopa/io-functions-commons/dist/src/models/profile";
+
+import { getConfigOrThrow } from "../utils/config";
+import { cosmosdbInstance } from "../utils/cosmosdb";
+
+import { ProfileToSanitize, sanitizeProfileEmail } from "./index";
+
+getConfigOrThrow();
+
+const profilesContainer = cosmosdbInstance.container(PROFILE_COLLECTION_NAME);
+
+const profileModel = new ProfileModel(profilesContainer);
+
+const createSanitizeProfileEmailsFunction = azureFunction(
+  H.of(sanitizeProfileEmail)
+);
+
+export default createSanitizeProfileEmailsFunction({
+  inputDecoder: ProfileToSanitize,
```

Comment:
What should happen in case the input does not satisfy the decoder?

Reply:
the function exits with a `ValidationError`, after five retries the malformed input will be moved to `profile-to-sanitize-poison` queue

(the same appens with unhandled errors in the controller)